pyeudiw/satosa/default/openid4vp_backend.py

```diff
@@ -443,3 +443,11 @@ def server_url(self):
         Returns the server url
         """
         return self._server_url
+
+def load_handlers(config: dict, trust_evaluator: CombinedTrustEvaluator):
```

Comment:
```suggestion
def load_handlers(config: dict, trust_evaluator: CombinedTrustEvaluator):
```

handlers of WHAT?

rename to load credential_presentation_handlers

why it is a spare function in the openid4vp backend?
why is it not a class methos? If it cannot be a class method, please move it in a utils file within this folder.

Reply:
[rev: fix semantic for method of load_credential_presentation_handlers](https://github.com/italia/eudi-wallet-it-python/pull/424/commits/994bb18271ac69fa3be33b5ca4a0cd0b62d76e52)